lucida/calendar/CalendarClient/CalendarClient.java

```diff
@@ -22,17 +31,19 @@
 public class CalendarClient {
 	public static void main(String [] args) 
 			throws IOException {
-		// Collect the port number.
-		Properties port_cfg = new Properties();
-		InputStream input = new FileInputStream("../../config.properties");
-		port_cfg.load(input);
-		String port_str = port_cfg.getProperty("CA_PORT");
-		Integer port = Integer.valueOf(port_str);
-		if (args.length == 1) {
-			port = Integer.parseInt(args[0]);
-		} else {
-			System.out.println("Using default port for Calendar Client: " + port);
+		// Get the port ID from Mongodb
+		String mongo_addr = "localhost";
```

Comment:
Add mongodb port and host (and maybe database name) to configuration or top of this file.